services/assistant/v2.js

```diff
@@ -17,10 +17,11 @@
 module.exports = function(RED) {
   const SERVICE_IDENTIFIER = 'assistant',
     OLD_SERVICE_IDENTIFIER = 'conversation',
-    SERVICE_VERSION = '2018-11-08';
+    SERVICE_VERSION = '2018-11-08',
+    AssistantV2 = require('ibm-watson/assistant/v2'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4882575)